libraries/AC_Fence/AC_Fence.h

```diff
@@ -12,19 +12,23 @@
 #define AC_FENCE_TYPE_ALT_MAX                       1       // high alt fence which usually initiates an RTL
 #define AC_FENCE_TYPE_CIRCLE                        2       // circular horizontal fence (usually initiates an RTL)
 #define AC_FENCE_TYPE_POLYGON                       4       // polygon horizontal fence
+#define AC_FENCE_TYPE_ALT_MIN                       8       // low alt fence which usually initiates an RTL
 
 // valid actions should a fence be breached
 #define AC_FENCE_ACTION_REPORT_ONLY                 0       // report to GCS that boundary has been breached but take no further action
 #define AC_FENCE_ACTION_RTL_AND_LAND                1       // return to launch and, if that fails, land
 #define AC_FENCE_ACTION_ALWAYS_LAND                 2       // always land
 #define AC_FENCE_ACTION_SMART_RTL                   3       // smartRTL, if that fails, RTL, it that still fails, land
 #define AC_FENCE_ACTION_BRAKE                       4       // brake, if that fails, land
+#define AC_FENCE_ACTION_GUIDED                      5       // guided mode, with target waypoint as fence return point
+#define AC_FENCE_ACTION_GUIDED_THROTTLE_PASS        6       // guided mode, but pilot retains manual throttle control
 
 // default boundaries
 #define AC_FENCE_ALT_MAX_DEFAULT                    100.0f  // default max altitude is 100m
+#define AC_FENCE_ALT_MAX_BACKUP_DISTANCE            20.0f   // after fence is broken we recreate the fence 20m further up
 #define AC_FENCE_ALT_MIN_DEFAULT                    -10.0f  // default maximum depth in meters
+#define AC_FENCE_ALT_MIN_BACKUP_DISTANCE            20.0f   // after fence is broken we recreate the fence 20m further down
 #define AC_FENCE_CIRCLE_RADIUS_DEFAULT              300.0f  // default circular fence radius is 300m
-#define AC_FENCE_ALT_MAX_BACKUP_DISTANCE            20.0f   // after fence is broken we recreate the fence 20m further up
```

Comment:
Random reordering?

Reply:
Thought it made more sense to group the max/min defaults together by type, but can swap them to be grouped as originally.